fe/src/main/java/org/apache/doris/alter/AlterOpType.java

```diff
@@ -0,0 +1,62 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package org.apache.doris.alter;
+
+public enum AlterOpType {
+    // rollup
+    ADD_ROLLUP,
+    DROP_ROLLUP,
+    // schema change
+    SCHEMA_CHANGE,
+    // partition
+    ADD_PARTITION,
+    DROP_PARTITION,
+    REPLACE_PARTITION,
+    MODIFY_PARTITION,
+    // rename
+    RENAME,
+    // table property
+    MODIFY_TABLE_PROPERTY,
+    MODIFY_TABLE_PROPERTY_SYNC, // Some operations are performed synchronously, so we distinguish them by suffix _SYNC
+    // others operation, such as add/drop backend. currently we do not care about them
+    ALTER_OTHER,
+
+    INVALID_OP;
+
+    // true means 2 operations have no conflict.
+    public static Boolean[][] CONFLICT_MATRIX;
```

Comment:
If true means 2 operations have no conflict. which should be `compatibilityMatrix`.

Reply:
OK